var/spack/repos/builtin/packages/python/package.py

```diff
@@ -224,6 +224,8 @@ class Python(AutotoolsPackage):
     conflicts('+tix', when='~tkinter',
               msg='python+tix requires python+tix+tkinter')
 
+    conflicts('%nvhpc')
```

Comment:
This is also really unfortunate, but not that surprising. We've also had a lot of problems with `python %intel` in the past

Reply:
I'd like to come back to it in the future.